recipes/jupyterlab-kernelspy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jupyterlab-kernelspy" %}
+{% set version = "3.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 557ff653901ef3c236fbfc549e55910ba3ffc9f63d7fbef51c75fa58772f2e4c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.7.9,<0.8
+  run:
+    - jupyterlab >=3.0,<4
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/jupyterlab-kernelspy/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\jupyterlab-kernelspy\\package.json (exit 0) else (exit 1)          # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlab-contrib/jupyterlab-kernelspy
+  summary: A JupyterLab extension for inspecting messages to/from a kernel
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - fcollonval
+    - jtpio
+    - vidartf
```

Comment:
cc @fcollonval @vidartf if you would like to be listed here

Reply:
Thanks @jtpio I agree to be maintainer on the conda recipe too.